fpga.chart.py

```diff
@@ -179,32 +181,37 @@ def _get_xilinx_fpga_power(self, idx):
 
     def _get_fpga_temp(self, idx):
         if os.path.exists(self.intel_cmd) and os.access(self.intel_cmd, os.X_OK):
```

Comment:
Proposal: determine on startup whether the commands exist and are executable. Store them into a variable and use that. Makes the code cleaner.

Reply:
I need to rethink this method of detection following on from the discussion about mixed cards.